selenium/util/web_driver_utils.rb

```diff
@@ -0,0 +1,77 @@
+require 'selenium-webdriver'
+require 'page-object'
+require 'json'
+
+class WebDriverUtils
+
+  @config = YAML.load_file File.join(ENV['HOME'], '/.collabosphere_selenium/settings.yml')
+
+  def self.driver
+    Selenium::WebDriver.for @config['webdriver'].to_sym
+  end
+
+  def self.base_url
+    @config['base_url']
+  end
+
+  def self.sub_account
+    @config['sub_account']
+  end
+
+  def self.page_load_wait
+    @config['page_load_timeout']
+  end
+
+  def self.page_update_wait
+    @config['ajax_timeout']
+  end
+
+  def self.admin_username
+    @config['admin_username']
+  end
+
+  def self.admin_password
+    @config['admin_password']
+  end
+
+  def self.test_user_password
+    @config['test_user_password']
+  end
+
+  def self.lti_key
+    @config['lti_key']
+  end
+
+  def self.lti_secret
+    @config['lti_secret']
+  end
+
+  def self.asset_library_url
+    @config['lti_asset_library_url']
+  end
+
+  def self.engagement_index_url
+    @config['lti_engagement_index_url']
+  end
+
+  def self.test_course_name(spec)
+    "#{spec.inspect.sub('RSpec::ExampleGroups::', '')}-#{Time.now.to_i.to_s}"
+  end
+
+  def self.load_test_users
+    test_users = File.join(ENV['HOME'], '/.collabosphere_selenium/testUsers.json')
```

Comment:
@nicolaasmatthijs we shouldn't put our actual test users file in the code, but we should definitely include an example file.  I'll add one to the folder.